src/_utils/composable/use-browser-location.ts

```diff
@@ -0,0 +1,67 @@
+import { onMounted, onUnmounted, Ref, ref } from 'vue'
+
+const hasWindow = typeof window !== 'undefined'
+const defaultWindow = hasWindow ? window : null
```

Comment:
```suggestion
const defaultWindow = typeof window !== 'undefined' ? window : null
```

Reply:
I think we should keep this function `hasWindow` and external it into its own utils module. So we can use it when we want to interact with the `window` object.